x/mongo/driver/mongocrypt/options/mongocrypt_context_options.go

```diff
@@ -63,3 +73,15 @@ func (eeo *ExplicitEncryptionOptions) SetAlgorithm(algorithm string) *ExplicitEn
 	eeo.Algorithm = algorithm
 	return eeo
 }
+
+// SetQueryType specifies the query type.
+func (eeo *ExplicitEncryptionOptions) SetQueryType(queryType *QueryType) *ExplicitEncryptionOptions {
+	eeo.QueryType = queryType
+	return eeo
+}
+
+// SetContentionFactor specifies the contention factor.
+func (eeo *ExplicitEncryptionOptions) SetContentionFactor(contentionFactor *int64) *ExplicitEncryptionOptions {
```

Comment:
Can the param type be `int64` instead of `*int64`?

Reply:
Done.